File: mirage/factories/instance.js
import { Factory, faker } from 'ember-cli-mirage';

export default Factory.extend({
  page() {
    return 1;
  },
  sequence(i) {
    return `${i}`;
  },
  text() {
    return faker.address.city();
  },
  note() {
    return null;
  },
  special() {
    return null;
  },
  flagged() {
    return false;
  }
});

File: spec/factories/instance.rb
FactoryBot.define do
  # sequence :sequence do |n|
  #   "#{n}"
  # end

  factory :instance do
    page { Faker::Number.digit }
    # sequence 2
    text { Faker::Address.city }
    added_on Time.now
    flagged false
    note { Faker::Lorem.sentence }
    special "Special"
    place
    user
    # book
  end

end
